en/docs/quick-starts/google-as-federated-authenticator-sample.md

```diff
@@ -0,0 +1,108 @@
+# Log into a Sample Application using Google
+
+This page guides you through using Google as a federated authenticator and logging into an application. Here, we use a **sample application** called Pickup. 
+
+----
+
+If you have your own application, click the button below.
+
+<a class="samplebtn_a" href="../../guides/federation/google" target="_blank" rel="nofollow noopener">I have my own application</a>
+
+----
+
+
+!!! tip " Before you begin"  
+    1.  [Create a Google domain](https://www.bettercloud.com/monitor/the-academy/create-google-apps-domain-three-easy-steps/).
+    2.  [Install](https://wso2.com/identity-and-access-management/install/) WSO2 Identity Server.
+	
+	3.  Navigate to `<IS_HOME>/bin` and run the product using one of the following commands. For more information, see [Running the product](insert-link)
+
+		```curl tab="Windows"
+		wso2server.bat
+		```
+
+		```curl tab="Linux/Mac OS"
+		wso2server.sh
+		```
+---
+
+## Set up Google as a SAML IdP
+
+1.  Access the Google Admin console by navigating to this URL:
+    <https://admin.google.com/>.
+2.  Log in using your administrator account.
+3.  Click **Admin Console**.
+4.  Click **Apps** > **SAML apps** from the left hand panel.
+	
+	!!! info 
+		If you do not see the Apps button on the home page, click **More
+		Controls** at the bottom of the page.
+
+	![more-controls-saml-apps](../../assets/img/samples/saml-app.png)
+	
+
+5.  Click on the 
+    ![more-controls-icon](../../assets/img/samples/more-controls.png) icon found at
+    the bottom-right of the page.
+6.  Click **SETUP MY OWN CUSTOM APP**  
+    ![setup-my-own-custom-app](../../assets/img/samples/set-up-my-own-app.png)
+    
+7.  Click **Download** next to the **IDP Metadata** field to download
+    the Google identity provider metadata.  
+    A `          GoogleIDPMetadata.xml         ` file is downloaded on
+    to your machine.  
+    
+    ![idp-metadata](../../assets/img/samples/download-idp-metadata.png)
+    
+8.  Click **Next** and enter an **Application Name** and **Description**
+    for your application. This is the name and description that your
+    users will see.  
+    You can also upload an image of your logo.
+9.  Click **Next** and enter the following details.
+
+    !!! info 
```

Comment:
this section is repeated in the guides page right? shall we make it a fragment?

Reply:
please check the same for the azure and twitter guides as well

Reply:
These are with values and images. The ones in the guides are general instructions.